Orange/widgets/utils/tests/test_itemmodels.py

```diff
@@ -528,5 +530,61 @@ def tip(i):
         self.assertEqual(tip(1), "footip")
 
 
+class TestTableModel(unittest.TestCase):
```

Comment:
I suggest that TestTableModel gets a test of SQL data too. Just to ensure that these two play well together.